server/conf/routes

```diff
@@ -162,8 +162,9 @@ GET     /applicants/:applicantId/files/:fileKey     controllers.FileController.s
 GET     /assets/*file               controllers.Assets.versioned(file)
 
 # Methods for applicants
-GET     /applicants/:applicantId/edit                                       controllers.applicant.ApplicantInformationController.setLangFromBrowser(request: Request, applicantId: Long)
-POST    /applicants/:applicantId                                            controllers.applicant.ApplicantInformationController.setLangFromSwitcher(request: Request, applicantId: Long)
+GET     /applicants/:applicantId/edit               controllers.applicant.ApplicantInformationController.setLangFromBrowser(request: Request, applicantId: Long)
```

Comment:
question (non-blocking): Did anything change about this route? I don't see anything different but github is marking it as changed.